### PR TITLE
docs(Spanish): add quick sort

### PR DESCRIPTION
**Pull request by brugner**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [x] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [x] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**
Add folder structure for Spanish translations and the explanation for the Quick Sort algorithm.

### CODE REVIEW

File: docs/es/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Algoritmos
+
+## Ordenamiento
+
+- [Quick Sort](./Ordenamiento/Quick-Sort.md)
+
+## Cadenas
+
+## Búsqueda
+
```
Comment (ming-tsai) [inline docs/es/README.md:10]: You could remove these due to we don't have the documentation about theses sections.
```suggestion
```

File: docs/es/Ordenamiento/Quick-Sort.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+**10. Vector ordenado:** [10, 30, 40, 50, 70, 80, 90]
+
+## Implementación
+
+- [Java](../../../algorithms/Java/sorting/quick-sort.java)
+- [JavaScript](../../../algorithms/JavaScript/src/sorting/quick-sort.js)
+- [Go](../../../algorithms/Go/sorting/quicksort.go)
+- [C++](../../../algorithms/CPlusPlus/Sorting/quick-sort.cpp)
+- [Python](../../../algorithms/Python/sorting/quicksort.py)
+
```
Comment (ming-tsai) [inline docs/es/Ordenamiento/Quick-Sort.md:51]: We have implementation of quick sort on `C` language.
```suggestion
- [C](../../../algorithms/C/sorting/quick-sort.c)
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @brugner,
There are some suggestions for the pull request.

**Comment by brugner** [comment]
Hi @ming-tsai, I've just commited your suggestions. Thanks!

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**